src/include/logging/dummy_log_manager.h

```diff
@@ -0,0 +1,49 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// dummy_log_manager.h
+//
+// Identification: src/backend/logging/loggers/dummy_log_manager.h
+//
+// Copyright (c) 2015-16, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include <string>
+
+#include "logging/log_manager.h"
+#include "common/macros.h"
+
+namespace peloton {
+namespace logging {
+
+class DummyLogManager : public LogManager {
```

Comment:
Again, what is a "dummy" manager used for? If it's only testing, then we should probably move it in into the tests dir.